src/content/examples/hi/15_Math_And_Physics/01_Soft_Body/description.mdx

```diff
@@ -1,12 +1,10 @@
 ---
 featuredImage: "../../../images/featured/15_Math_And_Physics-01_Soft_Body-thumbnail.png"
-title: Soft Body
-one-line-description: Simulate the physics of a soft body accelerating toward the mouse.
+title: कोमल शरीर
```

Comment:
Use same translation for soft across file

Reply:
ok, I will check this